docs/src/pages/TooltipsPage/TooltipsPage.js

```diff
@@ -58,6 +58,27 @@ const components = [
       </BpkTooltip>,
     ],
   },
+  {
+    id: 'dark',
+    title: 'Dark',
+    blurb: [<Paragraph>Tooltips can have a dark look too.</Paragraph>],
+    examples: [
+      <BpkTooltip
+        id="my-tooltip"
+        type={TOOLTIP_TYPES.dark}
+        target={
+          <Heading
+            level="h3"
+            className={getClassName('bpkdocs-tooltips-page__heading')}
+          >
+            DAR
```

Comment:
Should this say Dark or LHR?

Reply:
It's an airport code :P The other examples said LHR and EDI, so I added a third one

Reply:
It's just the last place I went https://en.wikipedia.org/wiki/Dar_es_Salaam

Reply:
Ah ok, can you update the hover line to change to this airport instead of London Heathrow, just for consistency?